components/chart/tooltip/shared.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Shared Tooltip
+page_title: Chart for Blazor | Shared Tooltip
+description: Shared Tooltip for the Blazor Chart
+slug: chart-tooltip-shared
+tags: telerik,blazor,chart,tooltip,shared,tooltips
+published: True
+position: 2
+---
+
+# Shared Tooltip for Telerik Blazor Chart
+
+The Telerik Chart allows you to show a unified tooltip for all categories in [Categorical]({%slug components/chart/databind%}#series-types) Charts.
+
+In this article:
+* [Basics](#basics)
+* [Customization](#customization)
+ * [Parameter Settings](#parameter-settings)
+ * [Shared Template](#shared-template)
+
+
+## Basics
+
+The shared tooltip provides summarized information of all data points from the hovered category. This tooltip will take precedence over tooltip settings defined for a specific series.
+
+To enable the shared tooltip:
+
+1. Include the `<ChartTooltip>` tag nested inside the `<TelerikChart>`.
+1. Set it's `Visible` parameter to `true`.
+1. Set it's `Shared` parameter to `true`.
+
+>caption Basic configuration of a Chart with Shared Tooltip
+
+````CSHTML
+@* This example shows how to enable a Shared Tooltip *@
+
+<TelerikChart>
+    <ChartTooltip Visible="true" Shared="true"></ChartTooltip>
+    <ChartSeriesItems>
+        <ChartSeries Type="ChartSeriesType.Bar" Name="Product 1" Data="@series1Data">
+        </ChartSeries>
+        <ChartSeries Type="ChartSeriesType.Bar" Name="Product 2" Data="@series2Data">
+        </ChartSeries>
+    </ChartSeriesItems>
+
+    <ChartCategoryAxes>
+        <ChartCategoryAxis Categories="@xAxisItems"></ChartCategoryAxis>
+    </ChartCategoryAxes>
+
+    <ChartTitle Text="Quarterly revenue per product"></ChartTitle>
+
+    <ChartLegend Position="ChartLegendPosition.Right">
+    </ChartLegend>
+</TelerikChart>
+
+@code {
+    public List<object> series1Data = new List<object>() { 10, 2, 5, 6 };
+    public List<object> series2Data = new List<object>() { 5, 8, 2, 7 };
+    public string[] xAxisItems = new string[] { "Q1", "Q2", "Q3", "Q4" };
+}
+````
+>caption The result from the code snippet above
+
+![shared tooltip example](images/shared-tooltip-basic.png)
```

Comment:
Could you take the screenshot again but with a smaller browser. The entire image could be about 600px wide, or even a bit less so it does not become blurry and constantly resize. Also, this will reduce its size.